packages/vscode/package.json

```diff
@@ -25,20 +25,24 @@
     ],
     "commands": [
       {
-        "command": "marko.showScriptOutput",
-        "title": "Marko: Debug: Show Extracted Script Output"
+        "command": "marko.debug.showScriptOutput",
+        "title": "Show Extracted Script Output",
+        "category": "Marko (Debug)"
```

Comment:
Small QOL improvement, could be split into a separate PR

<img width="597" height="136" alt="image" src="https://github.com/user-attachments/assets/541b12df-af6b-47d4-85fd-39b1109f8617" />